app/renderer/js/utils/enterprise-util.js

```diff
@@ -0,0 +1,53 @@
+const fs = require('fs');
+const process = require('process');
+
+const Logger = require('./logger-util');
+
+const logger = new Logger({
+	file: 'enterpise-util.log',
+	timestamp: true
+});
+
+let instance = null;
+
+class EnterpriseUtil {
+
+	constructor() {
+		if (instance) {
+			return instance;
+		}
+		instance = this;
+
+		this.reloadDB();
+		return instance;
+	}
+
+	reloadDB() {
+		let enterpriseJsonPath = null;
+		if (process.platform === 'win32') {
+			enterpriseJsonPath = 'C:\\Program Files\\Zulip\\enterprise_config.json';
```

Comment:
I would avoid using `Zulip` directory since if the admin installs a desktop app it ends up here. Also see the comment I made earlier (possible would be marked as outdated) about using path module and the file paths.

Reply:
Would be great if we can use the electron's userData api here to get the correct path of the installed app.